crowbar_framework/lib/crowbar/backup/restore.rb

```diff
@@ -163,7 +163,7 @@ def restore_chef
 
               record = JSON.load(file.read)
               filename = file.basename.to_s
-              if proposal?(filename) && type == :databags
+              if proposal?(filename) && !repo?(file) && type == :databags
```

Comment:
sounds good, but determination would also be by matching the file.to_s, or do you have another way? 

i could write it like this
```ruby
      def proposal?(file)
        !file.basename.to_s.match(/(_network\.json$)|(^template-(.*).json$)|(^queue\.json$)/) && \
          !crowbar_databag?(file)
      end

      def crowbar_databag?(file)
        file.to_s =~ /knife\/databags\/crowbar/
      end
```